Dockerfile.mysql

```diff
@@ -12,3 +12,12 @@ EXPOSE 3306
 
 # Set the data directory as a volume
 VOLUME /var/lib/mysql
+
+# Set the working directory
+WORKDIR /usr/src/app
+
+# Define the entry point
+ENTRYPOINT ["docker-entrypoint.sh"]
+
+# CMD to start MySQL server
+CMD ["mysqld"]
```

Comment:
Add EOF(End-of-file), need to add empty line at the end
![image](https://github.com/mate-academy/devops_todolist_docker_core_task_3_docker_compose/assets/87720583/e993ee21-8522-4a77-abdb-ed7488b1461d)